prime-router/src/test/kotlin/fhirengine/translation/NistElrTests.kt

```diff
@@ -0,0 +1,39 @@
+package gov.cdc.prime.router.fhirengine.translation
+
+import assertk.assertThat
+import assertk.assertions.isEqualTo
+import assertk.assertions.isNotEmpty
+import ca.uhn.hl7v2.DefaultHapiContext
+import ca.uhn.hl7v2.model.Message
+import ca.uhn.hl7v2.parser.CanonicalModelClassFactory
+import ca.uhn.hl7v2.util.Hl7InputStreamMessageIterator
+import ca.uhn.hl7v2.validation.impl.ValidationContextFactory
+import kotlin.test.Test
+import fhirengine.translation.hl7.structures.nistelr251.message.ORU_R01 as NIST_ELR_ORU_R01
+
+class NistElrTests {
+    @Test
+    fun `test nist elr java class`() {
+        val rawMessage = """
+MSH|^~\&|CDC PRIME - Atlanta, Georgia (Dekalb)^2.16.840.1.114222.4.1.237821^ISO|Avante at Ormond Beach^10D0876999^CLIA|PRIME_DOH|Prime ReportStream|20210210170737||ORU^R01^ORU_R01|371784|P|2.5.1|||NE|NE|USA||||PHLabReportNoAck^ELR_Receiver^2.16.840.1.113883.9.11^ISO
```

Comment:
I'm assuming this message contains all the possible fields the classes support? How was it created?

Reply:
I _think_ Jack grabbed it out of one of the elims messages, but I can take a second look and make sure it's comprehensive